tools/cli/go-whisk/whisk/client.go

```diff
@@ -67,14 +67,14 @@ type Client struct {
 }
 
 type Config struct {
```

Comment:
Can you align all the properties in `Config`?

Reply:
Done.